.travis.yml

```diff
@@ -62,89 +86,128 @@ matrix:
       env:
         - TEST_SLOW="true"
         - SPLIT="3/3"
-    - python: 3.4
+    - python: 3.5
       env:
         - TEST_SLOW="true"
         - SPLIT="1/3"
-    - python: 3.4
+    - python: 3.5
       env:
         - TEST_SLOW="true"
         - SPLIT="2/3"
-    - python: 3.4
+    - python: 3.5
       env:
         - TEST_SLOW="true"
         - SPLIT="3/3"
+
     - python: 2.7
       env:
-        - TEST_THEANO="true"
-      addons:
-        apt:
-          packages:
-          - libatlas-dev
-          - libatlas-base-dev
-          - liblapack-dev
-          - gfortran
-    - python: 3.4
-      env:
-        - TEST_THEANO="true"
+        - TEST_AUTOWRAP="true"
       addons:
         apt:
           packages:
-          - libatlas-dev
-          - libatlas-base-dev
-          - liblapack-dev
-          - gfortran
-    - python: 2.7
-      env:
-        - TEST_AUTOWRAP="true"
-      sudo: true
+            - gfortran
+            - gcc
+            - python-numpy
+            - cython
       virtualenv:
         system_site_packages: true
```

Comment:
Only the one set of builds use conda. There is no gmpy for conda, so we can't use it everywhere (and I don't have a good Linux building VM to build one myself).